src/stores/OverlayStore/OverlayStore.ts

```diff
@@ -1015,7 +1026,7 @@ export class OverlayStore {
     @observable colorbar: OverlayColorbarSettings;
     @observable beam: OverlayBeamSettings;
 
-    private constructor() {
+    public constructor(fullViewWidth?: number, fullViewHeight?: number, base: number = 1, defaultGap: number = 2) {
```

Comment:
The constructor can be set back to private and the input parameters are not used.